src/test/ui/const-generics/uninferred-consts-during-codegen-1.rs

```diff
@@ -0,0 +1,18 @@
+// run-pass
+
+#![feature(const_generics)]
+//~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash
+
+use std::fmt;
+
+struct Array<T, const N: usize>([T; N]);
+
+impl<T: fmt::Debug, const N: usize> fmt::Debug for Array<T, {N}> {
```

Comment:
`N` should also work, how are we tracking that?

Reply:
This is https://github.com/rust-lang/rust/issues/6080.